template/docker/Dockerfile.j2

```diff
@@ -23,9 +23,12 @@ RUN microdnf install -y yum \
   && yum clean all \
   && microdnf clean all
 
+# Install kerberos client libraries
+RUN microdnf install -y krb5-libs libkadm5 && microdnf clean all
+
 COPY LICENSE /licenses/LICENSE
 
-COPY --from=builder /app/stackable-{[ operator.name }] /
+COPY --from=builder /app/* /usr/local/bin
```

Comment:
This is technically breaking, since anyone assuming that the operator would be available in / will be broken... but as long as you relied in the ENTRYPOINT you should be fine.

Reply:
Fine for me in principle, but I have to admit I don't fully understand where the need for this change comes from?

Reply:
Ah, thanks!